google/cloud/spanner/row.h

```diff
@@ -206,6 +218,21 @@ constexpr std::string PromoteLiteralImpl(char const*);
 constexpr std::int64_t PromoteLiteralImpl(int);
 template <typename T>
 using PromoteLiteral = decltype(PromoteLiteralImpl(std::declval<T>()));
+
+// A helper functor to be used with `internal::ForEach` to iterate the columns
+// of a `Row<Ts...>` in the ParseRow() function.
+struct ExtractValue {
+  Status& status;
+  template <typename T, typename It>
+  void operator()(T& t, It& it) const {
+    auto x = it++->template get<T>();
+    if (x) {
```

Comment:
Optional: Switch sense to `if (!x)` for the reasons we chatted about this afternoon?

Reply:
Done.